### PR TITLE
Update fieldset to match the design spec

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Update fieldset to match the design spec

## QA
- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/examples/base/forms/fieldset/
- Check that the styles matches the [design spec](https://github.com/ubuntudesign/vanilla-design/blob/master/Forms/forms.md)

## Details
Fixes https://github.com/vanilla-framework/vanilla-framework/issues/1906

### CODE REVIEW

**Comment by kwm14** [comment]
Update looks good 👍

**Comment by barrymcgee** [comment]
@anthonydillon The padding on the fieldset is 8px but the code states two variables are used and right/left should be less than top/bottom - right?

screen shot 2018-07-24 at 12 18 35

File: scss/_base_forms.scss [hunk 1]
```diff
@@ -236,9 +236,11 @@ $input-margin-bottom: $spv-inter--scaleable - $spv-nudge * 2;
   // Fieldset styles
   fieldset {
     background-color: $color-light;
-    border: 0;
+    border: 1px solid $color-mid-light;
     border-radius: $border-radius;
     color: $color-dark;
+    padding: $spv-intra $sph-intra--condensed;
+    margin-bottom: $spv-intra * 3;
```
Comment (Caleb-Ellis) [inline scss/_base_forms.scss]: This should be `$spv-inter--scaleable` - it's 1.5rem by default but can be adjusted with `$multi`

**Comment by anthonydillon** [comment]
@Caleb-Ellis @barrymcgee updated the margin to scaleable.

File: scss/_base_forms.scss [hunk 2]
```diff
@@ -236,9 +236,11 @@ $input-margin-bottom: $spv-inter--scaleable - $spv-nudge * 2;
   // Fieldset styles
   fieldset {
     background-color: $color-light;
-    border: 0;
+    border: 1px solid $color-mid-light;
     border-radius: $border-radius;
     color: $color-dark;
+    padding: $spv-intra $sph-intra--condensed;
+    margin-bottom: $spv-inter--scaleable * 3;
```
Comment (Caleb-Ellis) [inline scss/_base_forms.scss]: can remove the `* 3` because now it's 4.5rem

**Comment by anthonydillon** [comment]
Does this mean that `$spv-inter` and `$spv-inter--scaleable` are different
default sizes?

**Comment by anthonydillon** [comment]
@Caleb-Ellis updated.

**Pull request merged**